fuzz/project.go

```diff
@@ -0,0 +1,6 @@
+package fuzz
+
+type Project struct {
+	Directory string
+	Quiet     bool
+}
```

Comment:
Maybe to avoid the too many files with not much content in the `fuzz` package we can put this important Project struct (in term of Domain) at the top of the `fuzz.go` (as `fuzz.go` is self defined as the entry point of the `fuzz` package where the main things are defined).

Then we can delete this file.